sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -51,6 +51,11 @@ def _get_match(proto, filter_fn):
   return query[0]
 
 
+# V1b3 MetricStructuredName keys to accept and copy to the MetricKey labels.
+WHITELISTED_LABELS = [
```

Comment:
Whitelist seems like the wrong name here, maybe V1b3_LABELS or STRUCTURED_NAME_LABELS?

Reply:
Done